NC-NEWS/src/components/Articles.jsx

```diff
@@ -0,0 +1,26 @@
+import getArticles from "../api";
+import ArticlesCard from "./ArticlesCard";
+import { useEffect, useState } from "react";
+
+const Articles = () => {
+  const [articlesData, setArticlesData] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticlesData(articles);
```

Comment:
Please add an `isLoading` pattern for when the data is available. Currently there's a blank screen while it fetches data, so it's not great for the user experience.